tla/consensus/ccfraft.tla

```diff
@@ -1356,7 +1354,8 @@ StateTransitionsProp ==
 
 PendingBecomesFollowerProp ==
     \* A pending node that becomes part of any configuration immediately transitions to Follower.
-    [][\A s \in { s \in Servers : state[s] = Pending } : 
+    \* TODO: this is not right, the transition only takes place once the node receives a message
```

Comment:
I think this is fine as the `configurations` still only include the node after it has received at AE with its own addition

Reply:
Ok, yes, agreed. I will rephrase the "become", I think it's "becomes aware that it is part of any configuration".